tripBackend/src/modules/productSchedule/queries/handlers/getProductScheduleByProductScheduleId.handler.ts

```
@@ -0,0 +1,36 @@
+	): Promise<HttpResponseBodySuccessDto<ProductScheduleEntity> | HttpException> {
+		const productSchedule =
+			await this.productScheduleRepository.findProductScheduleByProductScheduleId(
+				query.productScheduleId,
+				query.filter?.status,
+			);
+
+		if (!productSchedule) {
+			throw new NotFoundException('productScheduleId');
+		}
+
+		delete productSchedule.product.supplier;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)